tools/generateVendorTypeAndAliases.js

```diff
@@ -4,7 +4,7 @@
 
 const fs = require("fs");
 const Path = require("path");
-const snakeCase = require("../utils/snakeCase");
+const snakeCase = require("../src/snakeCase");
```

Comment:
I moved this file from `utils/snakeCase` directly into `src/`, so that the top-level `utils` directory could be removed.